### PR TITLE
fix(lint/wpt-tests-exist): make separate warnings

### PR DESCRIPTION
**Pull request by marcoscaceres**
Easier to work with: 

Screen Shot 2021-05-05 at 3 08 01 pm

### CODE REVIEW

**Review by sidvishnoi — APPROVED** [review]
Do check how it looks from command line.

**Comment by marcoscaceres** [comment]
Good point. Will do.

**Comment by marcoscaceres** [comment]
Not the worse... not the best...: 

Screen Shot 2021-05-05 at 5 00 03 pm

**Comment by marcoscaceres** [comment]
Is the above fixable to remove the HTML?

**Comment by sidvishnoi** [comment]
I think this will require updating the markdown renderer in respec2html.

**Comment by sidvishnoi** [comment]
I was also wondering if it would make sense to showWarning for each element with missing tests, instead of displaying unique tests across the document. Might help in hunting down bad tests.

**Comment by marcoscaceres** [comment]
> I was also wondering if it would make sense to showWarning for each element with missing tests, instead of displaying unique tests across the document. Might help in hunting down bad tests.

Yeah, that might be better... having a single warning for a bunch of broken things is not amazing.

**Comment by marcoscaceres** [comment]
ok, better... 

Screen Shot 2021-05-06 at 4 03 42 pm

**Review by sidvishnoi — APPROVED** [review]
Don't forget to update commit msg before merge.

File: src/core/linter-rules/wpt-tests-exist.js [hunk 1]
```diff
@@ -26,35 +27,22 @@ export async function run(conf) {
...
       .map(test => test.trim().split("#")[0])
-      .filter(test => test);
-
-    const missingTests = tests.filter(test => !filesInWPT.has(test));
-    if (missingTests.length) {
-      offendingElements.push(elem);
-      missingTests.forEach(test => offendingTests.add(test));
-    }
+      .filter(test => test && !filesInWPT.has(test))
+      .map(missingTest => {
```
Comment (sidvishnoi) [inline src/core/linter-rules/wpt-tests-exist.js:39]: Maybe group by element?

Comment (marcoscaceres) [reply src/core/linter-rules/wpt-tests-exist.js:39]: I prefer them as they are for now. But will update them after I play around bit.

**Pull request merged**